test/integration/verify-test.ts

```diff
@@ -4,63 +4,114 @@ const eventPayload = {
   foo: "bar",
 };
 const secret = "mysecret";
-const signature = "sha1=d03207e4b030cf234e3447bac4d93add4c6643d8";
+const signatureSHA1 = "sha1=d03207e4b030cf234e3447bac4d93add4c6643d8";
+const signatureSHA256 =
+  "sha256=4864d2759938a15468b5df9ade20bf161da9b4f737ea61794142f3484236bda3";
 
-test("verify() without options throws", () => {
-  expect(() => verify()).toThrow();
-});
+describe("throws when calling", () => {
+  test("verify() without options", () => {
+    expect(() => verify()).toThrow();
+  });
 
-test("verify(undefined, eventPayload) without secret throws", () => {
-  expect(() => verify.bind(null, undefined, eventPayload)()).toThrow();
-});
+  test("verify(undefined, eventPayload) without secret", () => {
+    expect(() => verify.bind(null, undefined, eventPayload)()).toThrow();
+  });
 
-test("verify(secret) without eventPayload throws", () => {
-  expect(() => verify.bind(null, secret)()).toThrow();
-});
+  test("verify(secret) without eventPayload", () => {
+    expect(() => verify.bind(null, secret)()).toThrow();
+  });
 
-test("verify(secret, eventPayload) without options.signature throws", () => {
-  expect(() => verify.bind(null, secret, eventPayload)()).toThrow();
+  test("verify(secret, eventPayload) without options.signature", () => {
+    expect(() => verify.bind(null, secret, eventPayload)()).toThrow();
+  });
 });
 
-test("verify(secret, eventPayload, signature) returns true for correct signature", () => {
-  const signatureMatches = verify(secret, eventPayload, signature);
-  expect(signatureMatches).toBe(true);
-});
+describe("for SHA1 signature", () => {
+  test("verify(secret, eventPayload, signature) returns true for correct signature", () => {
+    const signatureMatches = verify(secret, eventPayload, signatureSHA1);
+    expect(signatureMatches).toBe(true);
+  });
 
-test("verify(secret, eventPayload, signature) returns false for incorrect signature", () => {
-  const signatureMatches = verify(secret, eventPayload, "foo");
-  expect(signatureMatches).toBe(false);
-});
+  test("verify(secret, eventPayload, signature) returns false for incorrect signature", () => {
+    const signatureMatches = verify(secret, eventPayload, "foo");
+    expect(signatureMatches).toBe(false);
+  });
 
-test("verify(secret, eventPayload, signature) returns false for correct secret", () => {
-  const signatureMatches = verify("foo", eventPayload, signature);
-  expect(signatureMatches).toBe(false);
+  test("verify(secret, eventPayload, signature) returns false for correct secret", () => {
+    const signatureMatches = verify("foo", eventPayload, signatureSHA1);
+    expect(signatureMatches).toBe(false);
+  });
+
+  test("verify(secret, eventPayload, signature) returns true if eventPayload contains (#71)", () => {
```

Comment:
it looks like the test now exists twice? Did you mean to test with different algorithms? If so you need to change the test and maybe mention the algorithm in the test description? Also can you amend the description? I think it should have said
```suggestion
  test("verify(secret, eventPayload, signature) returns true if eventPayload contains special characters (#71)", () => {
```

Reply:
I forgot to change the algorithm there!🙏🏽 

For each unit test (sha1 and sha256) for special characters:
- [x] applying proper algorithm
- [x] mentioning algorithm in test description
- [x] amend `description()`
- [x] added `special characters` to the sentence